include/grpcpp/security/tls_credentials_options.h

```diff
@@ -104,6 +104,17 @@ class TlsCredentialsOptions {
   // version > 1.1.
   void set_crl_directory(const std::string& path);
 
+  // Sets the minimum TLS version that will be negotiated during the TLS
+  // handshake. If not set, the underlying SSL library will pick the version
+  // automatically.
+  // @param tls_version: The minimum TLS version.
+  void set_min_tls_version(grpc_tls_version tls_version);
+  // Sets the maximum TLS version that will be negotiated during the TLS
+  // handshake. If not set, the underlying SSL library will pick the version
+  // automatically.
+  // @param tls_version: The maximum TLS version.
+  void set_max_tls_version(grpc_tls_version tls_version);
+
```

Comment:
CPP need some unity tests and functional tests as well...

Reply:
For unity tests, you can probably put them under `grpc/test/cpp/server/credentials_test.cc` and `grpc/test/cpp/client/credentials_test.cc`. For e2e tests, see my comment below...

Reply:
I added the unit tests there. PTAL. Thanks.